docs/contributors/code/testing-overview.md

```diff
@@ -380,11 +380,11 @@ To locally run the tests in debug mode, follow these steps:
 
 ### Native mobile end-to-end tests
 
-Contributors to Gutenberg will note that PRs include continuous integration E2E tests running the native mobile E2E tests on Android and iOS. For troubleshooting failed tests, check our guide on [native mobile tests in continious integration](docs/contributors/native-mobile.md#native-mobile-e2e-tests-in-continuous-integration). More information on running these tests locally can be found in the [relevant directory README.md](https://github.com/WordPress/gutenberg/tree/HEAD/packages/react-native-editor/__device-tests__).
+Contributors to Gutenberg will note that PRs include continuous integration E2E tests running the native mobile E2E tests on Android and iOS. For troubleshooting failed tests, check our guide on [native mobile tests in continious integration](/docs/contributors/native-mobile.md#native-mobile-e2e-tests-in-continuous-integration). More information on running these tests locally can be found in the [relevant directory README.md](https://github.com/WordPress/gutenberg/tree/HEAD/packages/react-native-editor/__device-tests__).
```

Comment:
The missing leading slash wasn't necessarily causing issues, but based on the guidelines [here](https://github.com/WordPress/gutenberg/tree/trunk/docs/contributors/documentation#using-links) and the other links I see, always using a leading slash is the convention.

Reply:
Looks like the path of the document is wrong, the file `native-mobile.md` is located in [`/docs/contributors/code/native-mobile.md`](https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/native-mobile.md) (it's missing the `code` folder).

Reply:
I also found a typo: `continious` => `continuous`.

Reply:
Thanks for catching those two!

Reply:
Fixed now.